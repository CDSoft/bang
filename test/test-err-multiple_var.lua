generator(false)

var "foo" "foo"
var "bar" "bar"
var "baz" "baz"
var "foo" "baz" -- [test/test-err-multiple_var.lua:6] ERROR: var foo: multiple definition
