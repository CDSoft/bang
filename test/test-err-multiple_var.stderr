[test/test-err-multiple_var.lua:7] ERROR: var foo: multiple definition
